### DISCUSSION TITLE
Widgets

### DISCUSSION THREAD

**Discussion by VoltBringer** [Lissy93/dashy#1373] [Help and Support]
So I am quite the novice with anything outside of windows. I may have over complicated my setup but i am running a Proxmox VM of Ubunto -> Portainer - > Dashy. I cannot figure out how to add widgets into the code configuration and have scoured the interwebs for answers. Any help would be greatly appreciated.

**Comment by jcunix** [comment]
Volt, I ran into this as well, it's as simple as creating a section, editing the conf.yml file and adding the widgets: portion with the widgets under it, for example:

`  - name: Time
    widgets:
      - type: clock
        options: null
        timeZone: America/Indiana/Indianapolis
        format: en-GB
        hideDate: false
        id: 0_399_clock
      - type: weather
        options:
          apiKey: API
          city: City
          units: imperial
        hideDetails: false
        id: 1_399_weather
      - type: weather-forecast
        options:
          city: City
          numDays: 6
          apiKey: API
          units: imperial
        id: 2_399_weatherforecast
      - type: public-ip
        id: 3_399_publicip
    displayData:
      sortBy: default
      rows: 1
      cols: 1
      collapsed: false
      hideForGuests: false
    items: []
    filteredItems: []`

https://paste.ofcode.org/9XbdYJmBezEAKEuejqKXpt

Good luck!
